plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -595,6 +589,9 @@ private boolean onlyOnePluginIncluded() {
      * @return Update site Data
      */
     private UpdateSite.Data extractUpdateCenterData(Map<String, String> groupIDs){
+        if (config.getWar() != null) {
+            throw new IllegalStateException("Not to be used when -war is specified");
```

Comment:
Saw this getting thrown from https://github.com/jenkinsci/plugin-compat-tester/blob/7bb2fd846a1accf45e4983c1475d2a69ffa76070/plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java#L194-L200 though I am not sure why.